lib/plugins/package/lib/zipService.js

```diff
@@ -17,8 +18,15 @@ module.exports = {
       }
     });
 
+    // exclude the whole .serverless directory
     patterns.push('!.serverless/**');
```

Comment:
You have .serverless in the defaultExcludes https://github.com/serverless/serverless/blob/f816879baf983442e58dc3bc110e517e0e4cc695/lib/plugins/package/lib/packageService.js#L14. Is there a reason why explicitly add it here again? What if someone explicitly want's to add it?